superset/utils/core.py

```diff
@@ -1258,16 +1257,15 @@ def get_or_create_db(
 
 
 def get_example_database() -> "Database":
-    from superset import conf
-
-    db_uri = conf.get("SQLALCHEMY_EXAMPLES_URI") or conf.get("SQLALCHEMY_DATABASE_URI")
+    db_uri = (
```

Comment:
I'm kind of surprised Mypy didn't pick up on this, i.e., the `db_uri` could be `None` though the `get_or_create_db` is expecting a `str` rather than `Optional[str]`.